LayoutTests/platform/ios-15/TestExpectations

```diff
@@ -0,0 +1,12 @@
+# These are the layout test expectations for Apple's iOS port of WebKit (both WebKit1 and WebKit2).
+#
+# See http://trac.webkit.org/wiki/TestExpectations for more information on this file.
+
+#//////////////////////////////////////////////////////////////////////////////////////////
+# Platform-specific directories. Skipped here, then re-enabled on the appropriate platform.
+#//////////////////////////////////////////////////////////////////////////////////////////
+
+#test to verify expectations set here actually work
+mathml/presentation/mo-stretch.html [ Failure ]
+mathml/presentation/roots.xhtml [ Failure ]
+mathml/radical-fallback.html [ Failure ]
```

Comment:
Does this file need to be removed from the commit?

Reply:
I think it’s safe to leave it. I still need to figure out the re-base situation for those tests. But after this change lands I can then work out the rebases needed for IOS15 and iOS16.

Reply:
Ok. In that case, the comment above these expectations should change since it isn't a test.

Reply:
I have removed these expectations, and will add them back in a separate commit. Moving to land this.